### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Serhii-Leonenko**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,54 @@
...
+                         / self.distance_from_city_center, 1)
+
+    def serve_cars(self, list_of_cars: list):
+        income = 0
+        for car in list_of_cars:
+            if self.can_wash_car(car):
+                income += (car.comfort_class
+                           * (self.clean_power - car.clean_mark)
+                           * self.average_rating
+                           / self.distance_from_city_center)
```
Comment (TPolina) [inline app/main.py:42]: don't repeat yourself: use method you wrote

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,54 @@
...
+                car.clean_mark = self.clean_power
```
Comment (TPolina) [inline app/main.py:43]: same here

**Review by TPolina — CHANGES_REQUESTED** [review]

**Comment by TPolina** [comment]
fix flake8

File: app/main.py (hunk 1 above)
Comment (Serhii-Leonenko) [reply app/main.py:42]: But, if i don't mistake, it will be the best practice soon, or no ?)
https://prnt.sc/0nwHZjOTU9Bx

File: app/main.py (hunk 2 above)
Comment (TPolina) [reply app/main.py:45]: and here use `wash_single_car` method

**Review by TPolina — CHANGES_REQUESTED** [review]
please, fix remarks in the comments

File: app/main.py (hunk 1 above)
Comment (Serhii-Leonenko) [reply app/main.py]: aaaa, ok

File: app/main.py (hunk 2 above)
Comment (Serhii-Leonenko) [reply app/main.py:45]: yes, i understood you

File: app/main.py (hunk 1 above)
Comment (Serhii-Leonenko) [reply app/main.py]: Ok, i will try)

Comment (Serhii-Leonenko) [reply app/main.py]: Yes, you are right, unfortunately  i didn't think about that i can not to round the result of  serve_cars

**Review by TPolina — APPROVED** [review]

**Pull request closed without merging**